<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerStackLoadModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerStackLoadModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <widget class="QLabel" name="ilLable">
        <property name="text">
         <string>ILine:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkRangeWidget" name="ilRangeWidget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="xlLable">
        <property name="text">
         <string>XLine:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkRangeWidget" name="xlRangeWidget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="sampLabel">
        <property name="text">
         <string>Samples:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkRangeWidget" name="sampRangeWidget" native="true"/>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="uploadBtn">
        <property name="text">
         <string>upload</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
